### DISCUSSION TITLE
Why is there a line at bottom of login form ?

### DISCUSSION THREAD

**Discussion by chmike** [fomantic/Fomantic-UI#1333] [Elements (button, image, label, loader, ...)] [answered]
In the [login form layout example](https://fomantic-ui.com/examples/login.html), there is a horizontal line at the bottom below the login button. Why is it there ? How could I remove it ? 

![image](https://user-images.githubusercontent.com/235856/74814815-60587b80-52f8-11ea-9e62-d12431390200.png)

**Comment by exoego** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/127635/74815354-6e2ff000-533c-11ea-84f4-466e9a5e0321.png)

Do you mean this line ?

If so, this line is a part of visual styling of [`ui stacked segment`](https://fomantic-ui.com/elements/segment.html#stacked).

If you want remove `stacked` styling, just remove `stacked` class.
Then `ui segment` will look like below:
![image](https://user-images.githubusercontent.com/127635/74815486-ab947d80-533c-11ea-8cb0-eef65ca3bb1b.png)

**Comment by chmike** [comment]
Thank you. Where can I find the meaning or effect of the different classes ? What is the effect of the 'teal' class name for instance ?

**Comment by ColinFrick** [comment]
The documentation should cover this. For example all variation of segments are here: https://fomantic-ui.com/elements/segment.html
`teal` is a color modifier.

**Comment by chmike** [comment]
Sorry, I missed that documentation on stacked segments. It was clearly documented. Is there documentation of the color models ? I couldn’t find it. What other colors do I have as alternative to teal ?

**Comment by ColinFrick** [comment]
The different colors are in the documentation (f.e. for buttons https://fomantic-ui.com/elements/button.html#colored). You can also find all different colors in the [colors.less](https://github.com/fomantic/Fomantic-UI/blob/master/src/themes/default/globals/colors.less) file

**Comment by chmike** [comment]
This was very helpful. Thank you very much.
